components/raftstore/src/store/peer.rs

```diff
@@ -2963,6 +2963,15 @@ where
                         cp
                     ));
                 }
+                (ConfChangeType::RemoveNode, PeerRole::Voter)
+                    if kind == ConfChangeKind::Simple && current_voter.len() == 2 =>
+                {
+                    return Err(box_err!(
+                        "{} unsafe request {:?}, to remove voter directly from a 2 voters raft group, please demoting voter firstly. see issue 10595 for detail",
```

Comment:
How about leaving `issue 10595` in comment? It seems weird to referencing issue number in API resource.

Reply:
I don't think it's "unsafe" if it can only lead to unavailable. Unavailability is not safety.